doc/non-functional tests/security.md

```diff
@@ -0,0 +1,21 @@
+
+## Java
+
+A játék biztonságához az is nagy mértékben hozzájárul, hogy Java nyelven íródott. A Java Compiler más, nyelvekhez képest egész sok hibát képes elkapni. Mivel nem történik direkt memória elérés így valószínűtlen, hogy más alkalmazások és programok által használt memóriára hivatkozzunk. A beépített memória managementnek hála nem kell manuálisan felszabadítani a memóriát ezzel megint rengeteg sérülékenységet kizárva. A nyelv statikusan típusos így rengeteg típushibától meg tud kímélni minket. A hozzáférés módosítók plusz védelmet jelentenek a nem szándékos felülbírálás megakadályozásával. Futtatás előtt pedig egy bytecode-ellenőrzés végrehajtásra kerül, ami csökkenti a rosszindulatú kód futtatásának kockázatát. Ezeken felül még számos biztonsági funkcióval rendelkezik a Java, amiket nem sorolnék föl, de a fentiek alapján is látható, hogy a Java nyelv nagy hangsúlyt fektet a biztonságra, ez persze jelentős performancia veszteséggel és egyéb kompromisszumokkal jár.
```

Comment:
"Mivel nem történik direkt memória elérés így valószínűtlen": ide az "így" elé is kell vessző

Reply:
"statikusan típusos így rengeteg" itt is